### DISCUSSION TITLE
MD5 models, how to use

### DISCUSSION THREAD

**Discussion by AAS** [Novum/vkQuake#660] [Q&A] [answered]
Copied mg1 folder from rerelease started game with "vkquake -game mg1" changed r_md5models to 1, still no changes. 
What should I do to enable MD5 models?

**Comment by temx** [comment, marked as answer]
The MD5 models are in `id1`, not `mg1`. You need to use the rerelease's `id1` (single 172 MB `pak0.pak`), not the original id1 (18 & 33 MB `pak0.pak` + `pak1.pak`).

**Reply by AAS** [reply]
It worked, thank you!
